sdk/Sdk.Generators/MethodVisibilityChecker.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
+{
+    internal static class MethodVisibilityChecker
```

Comment:
Could this be defined as an extension class  i.e. `MethodSymbolExtensions` where  GetVisibility looks like this: `GetVisibility(this IMethodSymbol methodSymbol)`

Reply:
Moved to an extension method.